### DISCUSSION TITLE
Focusing Select items with custom content by typing

### DISCUSSION THREAD

**Discussion by Dremora** [ariakit/ariakit#2698] [Questions] [answered]
In a [Select example](https://ariakit.org/components/select), once the select is open, one can type a letter to focus an item that starts with that letter.

I have a [slightly modified version of this component](https://stackblitz.com/edit/hffy3v?file=select%2Fstyle.css,select%2Findex.tsx&theme=dark) that displays emojis in front of text. Typing to focus doesn't work anymore.

Is there any documentation on how this feature is implemented, and how to make it work with custom content?

**Comment by diegohaz** [comment, marked as answer]
Currently, there is no way to customize the typeahead behavior by hooking into it. However, you can disable the typeahead by setting `typeahead={false}` and handle the logic on your own. I converted this issue into a feature request (#2699)
